regolith/helpers/a_presentationhelper.py

```diff
@@ -82,6 +84,9 @@ def subparser(subpi):
                        help="The database that will be updated.  Defaults to "
                             "first database in the regolithrc.json file.",
                        )
+    subpi.add_argument("-nc", "--no_cal",
```

Comment:
this won't work, the shortcut has to be a single letter.   Just delete `-nc` so the user has to type the long name.